Sources/ClientRuntime/Networking/Http/SdkHttpRequest+CRT.swift

```diff
@@ -0,0 +1,62 @@
+//
```

Comment:
Remember: I'm not completely modularizing anything except event streams & event streams auth.

So don't sweat things that should be in HTTP, HTTPAuth, etc but didn't get moved.  But if something was moved to the _wrong_ module, we should address that now.

Reply:
`Sources/SmithyEventStreams/Header+CRT.swift ` is related to event streams headers, not HTTP headers.

So this file goes into `SmithyEventStreams` because:
- it is event stream related
- it has a CRT dependency (i.e. it can't go into `SmithyEventStreamsAPI`)